pgvectorscale/src/access_method/storage.rs

```diff
@@ -25,23 +25,24 @@ pub trait NodeDistanceMeasure {
         stats: &mut S,
     ) -> f32;
 }
-
 pub trait ArchivedData {
-    fn with_data(data: &mut [u8]) -> Pin<&mut Self>;
```

Comment:
i'm confused how ArchivedData is useful without with_data or another constructor.